lib/passport-verify-strategy.ts

```diff
@@ -142,8 +151,9 @@ export function createStrategy (
   verifyUser: (user: TranslatedResponseBody) => object | false,
   saveRequestId: (requestId: string, request: express.Request) => void,
   loadRequestId: (request: express.Request) => string,
-  serviceEntityId?: string
+  serviceEntityId?: string,
+  samlFormTemplateName?: string
```

Comment:
This way round means not breaking anything already using serviceEntityId. Although I believe that's only likely to be our stuff, and ideally they would go whichever one is most likely to be used first.